### PR TITLE
Update to Support Native API Changes from Verizon SE 1.2.0

### PR DESCRIPTION
**Pull request by lcsabaignat**

### CODE REVIEW

File: Verizon/MPVerizonNativeAdAdapter.m [hunk 1]
```diff
@@ -39,54 +40,73 @@ - (void)setupWithVASNativeAd:(VASNativeAd *)vasNativeAd
...
+            self.vasAdProperties[kVASVideoViewKey] = videoView;
+        }
     }
 }
 
--(void)dealloc
-{
-    MPLogTrace(@"Deallocating %@.", self);
-}
-
```
Comment (chauduyphanvu) [inline Verizon/MPVerizonNativeAdAdapter.m]: Is there any reason this gets removed?

File: Verizon/MPVerizonNativeCustomEvent.m [hunk 2]
```diff
@@ -29,11 +29,6 @@ - (id)init
     return self = [super init];
 }
 
--(void)dealloc
-{
-    MPLogTrace(@"Deallocating %@.", self);
-}
-
```
Comment (chauduyphanvu) [inline Verizon/MPVerizonNativeCustomEvent.m]: Is there any reason this gets removed?

File: Verizon/MPVerizonNativeAdAdapter.m (hunk 1 above)
Comment (lcsabaignat) [reply Verizon/MPVerizonNativeAdAdapter.m]: This was used for development testing and it squeezed itself into production. There is no need for them. We intended to remove it on https://github.com/mopub/mopub-ios-mediation/pull/202#discussion_r322465513, but somehow we missed it. Sorry for that.

File: Verizon/MPVerizonNativeCustomEvent.m (hunk 2 above)
Comment (lcsabaignat) [reply Verizon/MPVerizonNativeCustomEvent.m]: This was used for development testing and it squeezed itself into production. There is no need for them. We intended to remove it on https://github.com/mopub/mopub-ios-mediation/pull/202#discussion_r322465513, but somehow we missed it. Sorry for that.

**Comment by chauduyphanvu** [comment]
@lcsabaignat, thanks for the clarification!

Also, I wanted to check in on any progress you might have on the issue with the icon image asset from above.

**Comment by lcsabaignat** [comment]
@chauduyphanvu working on it, but unfortunately not yet, hopefully I'll figure it out by tomorrow

**Comment by lcsabaignat** [comment]
@chauduyphanvu Just pushed up a fix. The icon image should who up now.

**Pull request merged**